# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/tests/crashtests/1765667.html
# Commit: ec05c5d6c331
# Full Hash: ec05c5d6c331b5602f55214a1f08ff0a76f6ad8e
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-05-03 09:42:08
# Description:
#   Bug 1765667 - Clamp surface size of content tiles for backdrop-filter r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D145047
# ==============================================================================

diff -r 40b512ed4dbf -r ec05c5d6c331 gfx/tests/crashtests/1765667.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/tests/crashtests/1765667.html	Tue May 03 01:56:24 2022 +0000
@@ -0,0 +1,4 @@
+<style>
+* { backdrop-filter: drop-shadow(15em 17454.938982935513ex mediumpurple); }
+</style>
+